addons/event/views/event_event_views.xml

```diff
@@ -78,8 +78,7 @@
                                 <tree string="Communication" editable="bottom">
                                     <field name="sequence" widget="handle"/>
                                     <field name="notification_type"/>
-                                    <field name="template_model_id" invisible="1"/>
```

Comment:
Indeed, I see that the field is not removed there. So I was wondering if we should do it here, and if so, if we should also do it in the 15.0 branch https://github.com/odoo/odoo/pull/115881 ?

Reply:
I just noticed that your xml changes were in both 15.0 and 16.0 branches in the original fix, so I wondered if this change, in 16.0, should also be done in 15.0. But again, I leave this question open for you and reviewers.

Reply:
What Odoonan is saying is that in 15.0 you have kept this line https://github.com/odoo/odoo/pull/112527/files#diff-35ba5c308cdb8d8b3dda81053a97d9d12fa686b7fce6241a0db7629b87740b9dR235

So for consistency (to have the same code from 15.0 to master), it would be best to remove it here but also remove it in #115881 (15.0 and saas-15.2).

As a side note, we intended to keep theses lines originally just to make the change safer for XPATH (see conversation: https://github.com/odoo/odoo/pull/112527/files#r1113472150).